nomos-services/log/src/lib.rs

```diff
@@ -121,6 +158,7 @@ impl ServiceCore for Logger {
             }
             LoggerBackend::Stdout => tracing_appender::non_blocking(std::io::stdout()),
             LoggerBackend::Stderr => tracing_appender::non_blocking(std::io::stderr()),
+            LoggerBackend::Writer(writer) => tracing_appender::non_blocking(writer),
```

Comment:
Could you elaborate on the use case of this writer? For example, sending logs via network to fluentd?

Reply:
Main reason was because I wanted to collect logs in the chat app but at the same time control how those are printed to stdout. 
I think it could also be useful in tests where instead of dumping logs into a file and then reading you could directly read them on the fly.
This is general enough that it can be used in multiple ways

Reply:
I get it. Thank you for your comment!